egs/ifnenit/v1/run.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+stage=0
+nj=8
+color=1
+data_dir=data
+exp_dir=exp
+. ./path.sh
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. utils/parse_options.sh  # e.g. this parses the --stage option if supplied.
+
+
+numSilStates=4
+numStates=8
+num_gauss=10000
+
+numLeavesTri=500
+numGaussTri=20000
+
+
+
+if [ $stage -le 0 ]; then
+  # data preparation
+  echo "data preparation"
+  local/ienit_initialize.sh
+  local/prepare_data.sh
+fi
+
+if [ $stage -le 1 ]; then
+  # dict folder preparation
+  echo "dict folder preparation"
+  local/prepare_dict.sh
+  utils/prepare_lang.sh --num-sil-states 3 --num-nonsil-states 4 --position-dependent-phones false data/local/dict "<unk>" data/local/lang data/lang_nolm
+fi
+
+if [ $stage -le 2 ]; then
+  # LM preparation
+  echo "LM preparation"
+  cp -R $data_dir/lang_nolm -T $data_dir/lang
+  local/prepare_lm.sh --ngram 1 $data_dir/train/text $data_dir/lang || exit 1;
```

Comment:
Could you please (instead of using `local/prepare_lm.sh`) make the grammar (G.fst) using `utils/make_unigram_grammar.pl` (and `fstcompile --isymbols=$lang/words.txt --osymbols=$lang/words.txt`) and remove `local/prepare_lm.sh`?